### PR TITLE
[Hotfix] change make private modal text [OSF-6480]

### PR DESCRIPTION
**Pull request by leb2dg**
## Purpose

Currently, the confirmation modal when making a project/component private is the same as the confirmation modal when making a project/component public.

screen shot 2016-06-09 at 3 44 25 pm

## Changes

* add check for whether the project is public or private
* add check for whether the node is a project or component
* add message for changing project/component to private

screen shot 2016-06-10 at 9 29 38 am
screen shot 2016-06-10 at 9 30 16 am

## Side effects

None

## Ticket

https://openscience.atlassian.net/browse/OSF-6480

### CODE REVIEW

File: website/static/js/nodesPrivacy.js [hunk 1]
```diff
@@ -139,9 +143,10 @@ var NodesPrivacyViewModel = function(node, onSetPrivacy) {
             return MESSAGES.makeEmbargoPublicTitle;
         }
 
-
         return {
-            warning: 'Warning',
+            warning: self.parentIsPublic ?
+                self.isComponent ? 'Make component private' : 'Make project private' :
```
Comment (brianjgeiger) [inline website/static/js/nodesPrivacy.js]: On L111 you are casting the word 'project' and 'component into a boolean, and here you are checking the boolean to decide whether to use the word 'project' or 'component. You could probably simplify this significantly.

**Comment by brianjgeiger** [comment]
A little simplification and this should be ready to go. 🐧

**Comment by brianjgeiger** [comment]
Looks great. 🎉 🐧 🎉

**Comment by sloria** [comment]
The modal text will read "Public forks and registrations of this *project*..." for components, but that's minor. This is good to go.

**Pull request merged**